ExampleMod/Content/RecipeHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content
+{
+	// In this class we separate recipe related code from our main class
+	public static class RecipeHelper
+	{
+		// Here we've made a helper method we can use to shorten our code.
+		// This is because many of our recipes follow the same terminology: one ingredient, one result, one possible required tile
+		private static void MakeSimpleRecipe(Mod mod, string modIngredient, short resultType, int ingredientStack = 1, int resultStack = 1, string reqTile = null)
```

Comment:
Ah, by the way: I believe we want to generally move away from using string to get instances of loaded types. We could use this method as an example of generics as well.